src/api/rpc_api/state.rs

```diff
@@ -88,6 +93,23 @@ pub trait GetStorage {
 	) -> Result<Option<V>>;
 
 	/// Retrieve the keys with prefix with pagination support.
+	/// Call the RPC substrate storage_keys_paged, which limits the number of returned keys.
+	/// See https://github.com/paritytech/substrate/blob/9f6fecfeea15345c983629af275b1f1702a50004/client/rpc/src/state/mod.rs#L54
```

Comment:
Can you move this description `See https://github.com/paritytech/substrate/blob/9f6fecfeea15345c983629af275b1f1702a50004/client/rpc/src/state/mod.rs#L54`

to the bottom with only two backslashes? I don't think the link should show up in the function description